spec/data_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'yaml'
+require 'spec_helper'
+
+RSpec.describe 'data' do
+  it 'has the same keys across all locales' do
+    keys = Dir['_data/*/settings.yml'].map { |path| hash_keys(YAML.load_file(path)) }
+
+    expect(keys).not_to be_empty
+    keys.combination(2).each do |first, second|
+      expect(first).to match_array(second)
+    end
+  end
+
+  def hash_keys(hash, parent_keys: [], all_keys: [])
```

Comment:
This implementation built upon a similar one in the IdP:

https://github.com/18F/identity-idp/blob/3e4246d/spec/i18n_spec.rb#L245-L256